### ISSUE TITLE
uhh... how do i use this?

### ISSUE THREAD

**Issue by Outfit8TSB** [Lenni0451/ViaProxyMultiLaunch#1]
```
[18:17:45] [main/INFO] (ViaProxy) Usage: java -jar ViaProxy-3.3.3.jar | Starts ViaProxy in graphical mode if available
[18:17:45] [main/INFO] (ViaProxy) Usage: java -jar ViaProxy-3.3.3.jar config <config file> | Starts ViaProxy with the specified config file
[18:17:45] [main/INFO] (ViaProxy) Usage: java -jar ViaProxy-3.3.3.jar cli --help | Starts ViaProxy in CLI mode
```
if i put java -jar viaproxy-3.3.3.jar --serverjar server.jar it shows this.
how am i supposed to do to use this plugin?

**Comment by Lenni0451** [comment]
This plugin hasn't been updated in a while and is not compatible with the current version of ViaProxy.
I'm working on updating it and will post here when I'm done.

**Comment by Lenni0451** [comment]
I just released the `2.0` version which provides better support for the latest ViaProxy version.
It also uses a config file now for easier configuration.
You can find it on the [releases page](https://github.com/ViaVersionAddons/ViaProxyMultiLaunch/releases/tag/v2.0).

If you need more help on how to start/configure ViaProxy itself, check out the [ViaProxy readme file](https://github.com/ViaVersion/ViaProxy).
